docs/source/api-client-python/index.rst

```diff
@@ -14,5 +14,5 @@ provides a simple genome browser that pulls data from the Genomics API.
 The Python client does not currently use
 `Google's Python client library <https://developers.google.com/api-client-library/python/>`_.
 If you want to use the client library, the
-`method documentation <https://developers.google.com/resources/api-libraries/documentation/genomics/v1beta2/python/latest/>`_
+`method documentation <https://developers.google.com/resources/api-libraries/documentation/genomics/v1/python/latest/>`_
```

Comment:
Possibly the following would connect the two together, in order to entice users to take advantage of the new client library features:

Though the current implementation of the Python client is not based on the \`Google's Python client library \<https://developers.google.com/api-client-library/python/>`_, an \`API Reference \<https://developers.google.com/resources/api-libraries/documentation/genomics/v1/python/latest/>`_ is now available for accessing the Google Genomics API via the client library.